atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/ColumnValueDescription.java

```diff
@@ -216,10 +241,22 @@ public Class<?> getJavaObjectTypeClass() {
         return type.getJavaObjectClass();
     }
 
+    /**
+     * Legacy persister method; will fail for reusable persisters.
```

Comment:
Hmm, I think this is no longer true because we added the backcompat one
```suggestion
     * Legacy persister method.
```